conf/app.routes

```diff
@@ -1,5 +1,8 @@
 # microservice specific routes
 
 ->         /hmrc-frontend           hmrcfrontend.Routes
-GET        /                        uk.gov.hmrc.ukimauthcheckerfrontend.controllers.HelloWorldController.helloWorld
+GET        /                        uk.gov.hmrc.ukimauthcheckerfrontend.controllers.IndexController.onPageLoad
+GET        /input                   uk.gov.hmrc.ukimauthcheckerfrontend.controllers.InputController.onPageLoad
+GET        /result                  uk.gov.hmrc.ukimauthcheckerfrontend.controllers.ResultController.onPageLoad
+GET        /outage                  uk.gov.hmrc.ukimauthcheckerfrontend.controllers.OutageController.onPageLoad
```

Comment:
Probably best to reroute this as 'error' and ErrorController (though in fairness I think we can just get away with an ErrorView, I don't think we need to redirect to a controller on errors)